app/Domain/Auth/CurrentUser.php

```diff
@@ -28,6 +28,16 @@ public function name(): string
         return $this->user()->name;
     }
 
+    public function firstName(): string
+    {
+        return $this->user()->first_name;
```

Comment:
Did you try this with a regular user (not a social login)? I think these fields are only filled in for social users in our case because UiTiD itself (the custom code on top of Auth0) does not store that info in these fields afaik. So we should probably not rely on them

Reply:
I created an regular user and made the following changes to make it work for both type of users:
- Use `https://publiq.be/first_name` as the key to get the first name
- Add fallback to empty string when `last_name` is missing